app/src/main/java/com/xmartlabs/template/database/AppDataBase.kt

```diff
@@ -1,10 +1,10 @@
 package com.xmartlabs.template.database
 
-import com.raizlabs.android.dbflow.annotation.Database
+import android.arch.persistence.room.RoomDatabase
 
-@Database(name = AppDataBase.NAME, version = AppDataBase.VERSION)
-object AppDataBase {
-  //TODO: change db name
-  const val NAME = "template_database"
-  const val VERSION = 1
-}
+//@Database(
```

Comment:
Remove the commented code, or if it's gonna be used then add a TODO.

Reply:
I think todo is fine